src/utils/queries/clientQuery.js

```diff
@@ -1,7 +1,15 @@
 import ApolloClient from 'apollo-boost';
 
 const client = new ApolloClient({
-  uri: 'https://cors-anywhere.herokuapp.com/https://pure-castle-14648.herokuapp.com/graphql'
+  uri: 'https://cors-anywhere.herokuapp.com/https://pure-castle-14648.herokuapp.com/graphql',
+  request: (operation) => {
+    const token = localStorage.getItem('token');
```

Comment:
who knew local storage would come in handy again? 😂

Reply:
😂